src/main/java/de/retest/ui/descriptors/Attributes.java

```diff
@@ -128,7 +127,8 @@ public int size() {
 	public Attributes applyChanges( final Set<AttributeDifference> attributeChanges ) {
 		final MutableAttributes result = new MutableAttributes( this );
 		for ( final AttributeDifference attributeDifference : attributeChanges ) {
-			if ( Objects.equal( attributes.get( attributeDifference.getKey() ), attributeDifference.getExpected() ) ) {
+			if ( ObjectUtil.equal( attributes.get( attributeDifference.getKey() ),
```

Comment:
There is https://docs.oracle.com/javase/8/docs/api/java/util/Objects.html#equals-java.lang.Object-java.lang.Object-